base.py

```diff
@@ -498,10 +498,21 @@ def update_time(scene):
 class MBDynReadLog(bpy.types.Operator):
     """ Imports MBDyn nodes and elements by parsing the .log file """
     bl_idname = "animate.read_mbdyn_log_file"
-    bl_label = "MBDyn .log file parsing"
```

Comment:
I agree in principle but it still seems like a dirty trick to me. 

Let's take a different approach: we can use a single global Enum property, adding it to the collection `MBDynSettingsScene`, for example. The possible values are `DO NOTHING`, `HIDE` and `DELETE`. 

Then we just warn the users (using the standard `report` method) that some nodes were removed from the model. They can change the behaviour for the next time the `.log` file is loaded.

This leads me to another feature that I'd like to add to to Blendyn: collecting all the `report` messages into a  log file (that could be just relative to the current session) that is written in the Text Editor... But I'm going off to a tangent here :)